tests/test_utils.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from bonfire.utils import (
+    split_equals,
+    validate_time_string,
+    get_version,
+    hms_to_seconds
+)
+
+
+@pytest.mark.parametrize(
+    "list_of_str, expected",
+    [
+        (
+            ["t1=test1", "t2=test2"],
+            {"t1": "test1", "t2": "test2"}
+        ),
+        (
+            ["company=redhat", "team=Dev Prod"],
```

Comment:
Hmmm actually, I don't think we should allow whitespace :laughing: you may have identified a tweak we should make to the regex here

Reply:
Haha, that's funny! I'll fix the test.